samples/Basic/Desktop/SkiaSharpSample/Class1.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using SkiaSharp;
+
+namespace SkiaSharpSample
+{
+	public class Class1 : SKDrawable
```

Comment:
This might be better served in the tests. If you need to check what was drawn, you can do something like I do and just check a few pixels' color.

Reply:
That's correct. I onlky use this for fast test, if all could be compiled. I could remove this later.